lib/home_page.dart

```diff
@@ -35,18 +35,25 @@ class _HomePageState extends State<HomePage> {
     SupabaseManager.supabaseClient.channel('public:messages').on(
       RealtimeListenTypes.postgresChanges,
       ChannelFilter(event: 'INSERT', schema: 'public', table: 'messages', filter: 'sender_id=neq.$profileId'),
+      // ignore: always_specify_types
```

Comment:
By resolving, I don't mean "ignoring". I'm sure these type ones are especially easily fixable. And for nested Json down below, we already have the `deep_pick`package: https://pub.dev/packages/deep_pick